src/test-run/index.js

```diff
@@ -249,3 +278,22 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
 };
+
+ServiceMessages[CLIENT_MESSAGES.executePendingBrowserWindowCommand] = async function (msg) {
+    this.debugLog.driverMessage(msg);
+
+    var command        = this.pendingBrowserWindowCommand;
+    var executionError = null;
+
+    try {
+        if (command.type === COMMAND_TYPE.resizeWindow)
+            await resizeWindow(msg.pageUrl, command.width, command.height);
+        else if (command.type === COMMAND_TYPE.resizeWindowToFitDevice)
+            await resizeWindow(msg.pageUrl, command.device, command.orientation);
+    }
+    catch (err) {
+        executionError = new ActionUnsupportedDeviceTypeError();
```

Comment:
I don't quite understand. Do you want to browser-natives add some html-formating in error message to testcafe can decorate it (for `deviceName` for example)? All browser-natives messages here https://github.com/DevExpress/testcafe-browser-natives/blob/master/src/messages.js 
Or we just need to create new category 'BrowserNativesError' and add our text to errors resieved from browser-natives?